.github/workflows/sycl_post_commit.yml

```diff
@@ -3,23 +3,27 @@ name: SYCL Post Commit
 on:
   push:
     branches:
-      - sycl
+    - sycl
+  pull_request:
+    branches:
+    - sycl
+    paths:
+    - .github/workflows/sycl_post_commit.yml
+
```

Comment:
Are you going to remove this before the merge?

Reply:
I don't think so. If these lines were present in the first place, this regression wouldn't happen.